r/DESCRIPTION

```diff
@@ -2,25 +2,27 @@ Package: rworker
 Title: What the Package Does (One Line, Title Case)
 Version: 0.0.0.9000
 Authors@R: 
-    person(given = "First",
-           family = "Last",
-           role = c("aut", "cre"),
-           email = "first.last@example.com",
+    person("First", "Last", , "first.last@example.com", role = c("aut", "cre"),
```

Comment:
styler?

Reply:
used [usethis::use_tidy_description()](https://usethis.r-lib.org/reference/tidyverse.html#:~:text=use_tidy_description()%3A%20puts%20fields%20in%20standard%20order%20and%20alphabetises%20dependencies.) and it did this. happy to change it back though.